packages/enhanced-img/package.json

```
@@ -45,9 +45,13 @@
 		"@types/node": "^18.19.3",
 		"estree-walker": "^3.0.3",
 		"rollup": "^4.14.2",
-		"svelte": "^4.2.10",
+		"svelte": "^4.0.0 || ^5.0.0-next.0",
```

this shouldn't change. this specifies the single version for development that we use while `peerDependencies` specifies all versions we allow from users
```suggestion
		"svelte": "^4.2.10",
```

---

Awesome, today I learned. I've pushed your suggestions to the PR now. I also updated the lockfile along the way as CI was complaining about it.